integrations/access/accessrequest/app.go

```diff
@@ -389,6 +466,42 @@ func (a *App) getMessageRecipients(ctx context.Context, req types.AccessRequest)
 	return recipientSet.ToSlice()
 }
 
+func (a *App) recipientsFromAccessMonitoringRules(ctx context.Context, req types.AccessRequest) *common.RecipientSet {
+	log := logger.Get(ctx)
+	recipientSet := common.NewRecipientSet()
+
+	// This switch is used to determine which plugins we are enabling access monitoring notification rules for.
+	switch a.pluginType {
+	// Enabled plugins are added to this case.
+	case types.PluginTypeSlack, types.PluginTypeOpsgenie:
```

Comment:
I don't want to expand the scope if Slack and OpsGenie are the two plugins you tested this with, but can you please make a note to add Mattermost here shortly after? Should be same as Slack?

Reply:
Removed the opsgenie one from this PR